### ISSUE TITLE
file share to nextcloud app crashes when folder does no longer exist

### ISSUE THREAD

**Issue by cocanut** [nextcloud/android#11716]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. share a file from another app to nextcloud app
2. nextcloud app will open the file exp lorer. Navigate to some folder like /foo/bar and upload the file to it
3. within nextcloud app delete the folder /foo/bar 
4. share another file from an app to nextcloud app
5. nextcloud app will open the folder the last file was shared to even though the folder no longer exists.| can't upload to that folder, neither can \ navigate back via the "<-" Button. When I try this I get the Error:

### Cause of error
```java
Exception in thread "Thread-9" java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getRemotePath()' on a null object reference
    at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:230)
    at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:399)
    at java.lang.Thread.run(Thread.java:1012)
```

### Expected behaviour

When the folder was deleted nextcloud app should start in the root folder /.

### Cause of error
```java
Exception in thread "Thread-9" java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getRemotePath()' on a null object reference
    at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:230)
    at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:399)
    at java.lang.Thread.run(Thread.java:1012)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30240290`
* Build flavor: `gplay`

### Device information
* Brand: `samsung`
* Device: `gta4xlwifi`
* Model: `SM-P610`
* Id: `TP1A.220624.014`
* Product: `gta4xlwifieea`

### Firmware
* SDK: `33`
* Release: `13`
* Incremental: `P610XXS3FWD2`

### Android version

13

### Device brand and model

gta4xlwifi

### Stock or custom OS?

Stock

### Nextcloud android app version

30240290

### Nextcloud server version

26.0.2

### Using a reverse proxy?

I don't know

**Comment by joshtrichards** [comment]
Duplicate of #11207

**Comment by joshtrichards** [comment]
Fixed in #11410

**Comment by AlvaroBrey** [comment]
Duplicate of #11207
